patches/cxs+6.2.0.patch

```diff
@@ -1,7 +1,16 @@
 diff --git a/node_modules/cxs/dist/index.js b/node_modules/cxs/dist/index.js
-index 2fc345c..45d39ef 100644
+index 2fc345c..ea8edc0 100644
 --- a/node_modules/cxs/dist/index.js
 +++ b/node_modules/cxs/dist/index.js
+@@ -1,7 +1,7 @@
+ 'use strict';
+ 
+ var cache = {};
+-var prefix = 'x';
++var prefix = 'edo-embedded-';
```

Comment:
Is this a yarn patch?

Important that the v3 library keeps working exact same, it's in use on many sites eg https://karenwomen.org/, seems likely this might break v3.

Any way we could fix this without editing this file?

Reply:
this should not affect v3, because we redefine the prefix for v3. 
https://github.com/everydotorg/donate-button/blob/c08ab0e2d2f4380850c1986014fbffbc99936614/packages/donate-button-next/src/components/embed-button/index.tsx#L6 https://github.com/everydotorg/donate-button/blob/c08ab0e2d2f4380850c1986014fbffbc99936614/packages/donate-button-next/src/components/widget/index.tsx#L36
you can check that v3 is still working in the preview:  [Visit Preview](https://donate-button-git-fork-fishbakh-n-nikita-bug-3b29d4-everydotorg.vercel.app/)

The main reason I made this change is because this prefix is applied in the wrong order, not quite sure why. I assume it's because of the nesting and import order. Anyway, I don't like the `cxs` package, but I couldn't find a better lightweight alternative that supports shadow dom out of the box.

Reply:
Do you think we need a package at all - could we switch to using css files? https://tinloof.com/blog/using-css-files-in-web-components